modules/ztp-definition-of-done-for-ztp-installations.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="ztp-definition-of-done-for-ztp-installations_{context}"]
+= Definition of done for ZTP installations
+
+In previous versions, the method to check for a completed zero touch provisioning (ZTP)
+installation is to directly query the spoke clusters for all of the following criteria:
+
+The expected additional MachineConfig objects are defined on the clusters:
+
+* sctp-module
+** workload-partitioning
+** disable-chronyd
+** container-mount-namespace
+** performance- (sufficient match for PAO machine config)
+* When all of the MachineConfig objects are defined, wait for the main MachineConfigPool to reach the updated state and remain in that state for two minutes.
+* The SRIOV network Operator deployment is in the `Available` state.
+* The kube-apiserver pod is in the `Ready` state.
+
+Performing all of these checks on each node can be a time-consuming process.
+This version of zero touch provisioning (ZTP) simplifies this process by adding a {rh-rhacm-first}
+static validator inform policy.
+This policy captures the existing criteria for a completed installation and validates that it moves
+to a compliant state only when ZTP provisioning of the spoke cluster is complete.
+
+ZTP uses the `PolicyGenTemplate` custom resource (CR) to define {rh-rhacm} policies that are
```

Comment:
Passive voice with a known subject: maybe "...that the PolicyGenerator calls to create the policies."

Reply:
change made